epsagon/utils.py

```diff
@@ -19,6 +21,16 @@
 }
 
 
+def get_env_or_val(env_key, value):
```

Comment:
you can use `x = os.getenv(env_key) or value`

Reply:
so no need for this function at all

Reply:
Done.